app/views/admins/_user_entry.haml

```diff
@@ -1,70 +1,85 @@
-
 %li.user.media
-  %div.pull-left
-    - if user.person
-      %span.media-object
-        = person_image_tag(user.person, size: :thumb_small)
+  .media-body.row
+    .col-md-1
+      - if user.person
+        %span.media-object
+          = person_image_tag(user.person, size: :thumb_small)
+    .col-md-11
+      .row
+        .col-md-7
+          %h4.media-heading
+            = user.try(:person).try(:name)
+        .col-md-5
+          .pull-right
+            %span.label.label-default
+              = t(".id")
+              = user.id
+            %span.label.label-info
+              = t(".guid")
+              = user.try(:person).try(:guid)
```

Comment:
You could have try locally to switch Ruby to 2.1 and run rspec but Travis will tell us soon ;)